certbot/docs/using.rst

```diff
@@ -298,9 +298,9 @@ There are also a number of third-party plugins for the client, provided by
 other developers. Many are beta/experimental, but some are already in
 widespread use:
 
-================== ==== ==== ===============================================================
+================== ==== ==== =================================================================
```

Comment:
Is there a reason the lines defining the table were adjusted? I'd rather keep commit history clean if not.

Reply:
Yes, the row I've added is 2 characters longer than the previous longest row.